calcom/cal.com
# Revamp Google Cal warning for Meet, Amie, and Vimcal

## What does this PR do?



This PR redesigns the "Requires Google Calendar" warning for Meet, Amie, and Vimcal

Fixes #7254, fixes #7029

## Current
<img width="495" alt="image" src="https://user-images.githubusercontent.com/65426560/220999784-1ecf3ef7-60b4-4d9d-bb66-d763e971508d.png">


## With no GCal connected
<img width="495" alt="image" src="https://user-images.githubusercontent.com/65426560/220999633-261af572-ade6-49e2-854c-0ad97b3ca7de.png">


## With GCal connected
<img width="495" alt="image" src="https://user-images.githubusercontent.com/65426560/220999560-9e606a86-45af-45a8-b942-2ce9a9630ab0.png">


Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Ensure that the "requires GCal" warning appears on the Meet, Vimcal, and Amie app pages
    - Make sure that the install button is disabled
- Add a GCal credential
- You should be able to install the previously blocked apps

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
